config/stargate.json

```diff
@@ -26,6 +26,7 @@
     "goerli": "0x7612aE2a34E5A363E137De748801FB4c86499152",
     "bsc-testnet": "0x0000000000000000000000000000000000000000",
     "lineatest": "0x0000000000000000000000000000000000000000",
+    "linea": "0x0000000000000000000000000000000000000000",
```

Comment:
same here...why do we add address 0 if we dont deploy the facet on this network?

Reply:
Same as above